x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/geo_threshold/index.ts

```diff
@@ -16,6 +16,9 @@ export function getAlertType(): AlertTypeModel<GeoThresholdAlertParams, AlertsCo
     name: i18n.translate('xpack.triggersActionsUI.geoThreshold.name.trackingThreshold', {
       defaultMessage: 'Tracking threshold',
     }),
+    description: i18n.translate('xpack.triggersActionsUI.geoThreshold.descriptionText', {
+      defaultMessage: 'Alert when a coordinate enters or leaves a geo boundary',
```

Comment:
Maps has been using the word "entity" for context like this. Here is an example in the documentation - https://www.elastic.co/guide/en/kibana/7.9/maps-top-hits-aggregation.html.

So how about "Alert when an entity enters or leaves a geo boundary"

Reply:
Thanks @nreese! I went ahead and updated the description in this commit: https://github.com/elastic/kibana/pull/81850/commits/577bace7102ae918ec7562dc1460a5b901c27145.